main.go

```diff
@@ -66,6 +67,8 @@ func main() {
 
 	version := flag.Bool("version", false, "Display the version and exit")
 
+	debug := flag.Bool("enable-debug-endpoint", false, "Enable debug endpoint")
```

Comment:
Remove the `enable` prefix: `--debug-endpoint`

Reply:
Should we call this --enable-debugging-handlers instead similar to the flag supported by kubelet?
Ref: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/#options

Reply:
sounds good to me

Reply:
Updated